app/townsquare/views.py

```diff
@@ -472,6 +472,8 @@ def api(request, activity_id):
             counter += 1; results[counter] += time.time() - start_time; start_time = time.time()
             comment_dict['sorted_match_curve'] = comment.profile.matchranking_this_round.sorted_match_curve if comment.profile.matchranking_this_round else None
             counter += 1; results[counter] += time.time() - start_time; start_time = time.time()
+            if comment.is_edited:
```

Comment:
is this in any point making a validation to know if the user editing the comment is actually the comment owner? worry about people editing others comments

Reply:
It's simply to add a label right ? Should be alright

Reply:
@octavioamu Actually, I completely missed this. This not actually validating things. I will add that quickly.

Reply:
Actually, never mind, @octavioamu. @thelostone-mc had written the validation intially for the `comment_v1` views endpoint at: 

https://github.com/gitcoinco/web/blob/3b0b1e1a3daf07739c4e683d781c5e9e11ee8405/app/townsquare/views.py#L589-L594

So the comment.is_edited is for the label as @thelostone-mc said @octavioamu. The endpoint has validation to make sure a user does not edit except their own comments.